framework/decode/vulkan_virtual_swapchain.cpp

```diff
@@ -608,8 +608,9 @@ VkResult VulkanVirtualSwapchain::AcquireNextImageKHR(VkResult                  o
     }
 
     result = func(device, swapchain, timeout, semaphore, fence, image_index);
-    if (result != VK_SUCCESS)
+    if ((result != VK_SUCCESS) && (result != VK_SUBOPTIMAL_KHR))
```

Comment:
I don't think we want to start failing for this result code.  `SUBOPTIMAL` isn't a failure.  If you'd like, you could put in a TODO comment referencing https://github.com/LunarG/gfxreconstruct/issues/759 but take out these tests of `result`.

Reply:
Yes, VK_SUBOPTIMAL_KHR isn't a failure.

If the test is only (result != VK_SUCCESS),  VK_SUBOPTIMAL_KHR value will match it and proceed the following error output:
        GFXRECON_LOG_ERROR("Virtual swapchain failed AcquireNextImageKHR 0x%08x for swapchain (ID = %" PRIu64 ")",
                           result,
                           swapchain_info->capture_id);
                           
That's why the test (result != VK_SUBOPTIMAL_KHR) is added.

Reply:
Oh!  I read this change as the opposite of what you've actually added.  Yes, this is correct.